modules/cli/src/main/scala/scala/cli/ScalaCli.scala

```diff
@@ -117,7 +117,7 @@ object ScalaCli {
     try main0(args)
     catch {
       case e: Throwable if !isCI && !printStackTraces =>
-        val workspace = CurrentParams.workspaceOpt.getOrElse(os.pwd)
+        val workspace = logDir(CurrentParams.workspaceOpt.getOrElse(os.pwd))
```

Comment:
If I'm not mistaken, can just be simplified to:
```suggestion
        val workspace = CurrentParams.workspaceOpt.filter(os.isDir).getOrElse(os.pwd)
```

then the `logDir` function is redundant.

Reply:
Yes, much better.